README.md

```diff
@@ -1,2 +1,4 @@
 # Git-Demo-Repo
 A repo to demonstrate git workflow
+Like this
```

Comment:
fix grammar here. Also no need of new line

Reply:
Sure